### DISCUSSION TITLE
Retrain an existing model with new data of the same entity, and add new entities. Spacy v3

### DISCUSSION THREAD

**Discussion by berrodriquez26** [explosion/spaCy#11039] [Help: Coding & Implementations] [answered]
Hello. I am making a model with 10 different entities, and I don´t want to train all the model at once. The problem is that I haven´t found anything useful on the internet. Can someone explain me step by step how to retrain an existing model in spacy using the version 3 please. There are a lot of information but spacy version 2 is being used.

**Comment by berrodriquez26** [comment]
There are similar questions. 
https://github.com/explosion/spaCy/discussions/8513
https://github.com/explosion/spaCy/discussions/8176
But I want a step by step guide to re train the model. Also, I know it is better to train all the model at once, but I want to do it in parts.

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @berrodriquez26,

The best way to see how retraining a NER model works is to check the [`ner_demo_update`](https://github.com/explosion/projects/tree/v3/pipelines/ner_demo_update) project. If you want a step-by-step guide, you can check each step [in the project's `project.yml` file](https://github.com/explosion/projects/blob/v3/pipelines/ner_demo_update/project.yml) workflows. From there, you should be able to pattern the project on your own dataset / use case. 

If you're not familiar on how spaCy Projects work, feel free [to check the documentation](https://spacy.io/usage/projects). You only need to install spaCy and you'll have access to the `spacy project` command. You can also clone the said `ner_demo_update` project using [`spacy project clone`](https://spacy.io/usage/projects#clone).
